scripts/genproto.sh

```diff
@@ -97,6 +117,19 @@ for pb in api/etcdserverpb/rpc server/etcdserver/api/v3lock/v3lockpb/v3lock serv
     Documentation/dev-guide/apispec/swagger/"${swaggerName}".swagger.json
 done
 
+# We only upgraded grpc-gateway from v1 to v2, but keep gogo/protobuf as it's for now.
+# So we have to convert v1 message to v2 message. Once we get rid of gogo/protobuf, and
+# start to depend on protobuf v2, then we can remove this patch.
+for pb in api/etcdserverpb/rpc server/etcdserver/api/v3lock/v3lockpb/v3lock server/etcdserver/api/v3election/v3electionpb/v3election; do
+  gwfile="$(dirname ${pb})/gw/$(basename ${pb}).pb.gw.go"
+
```

Comment:
Those sed operations are a little hard to read. Could you leave a comment that lists summarizes changes need to make v1 message compatible to v2? From what I see:
* add import of github.com/golang/protobuf/proto
* wrap message into protov1.MessageV2

Reply:
Good call. I will add comment to clearly explain this.

Reply:
Done.